GameServerSource/Assets/Packages/ExampleMessages/Mediators/NewPromoExampleMediator.cs

```diff
@@ -28,7 +25,7 @@ public class TitleNewsItemMessage : MessageBase
         public string Body;
         public override void Serialize(NetworkWriter writer)
         {
-            var json = PlayFab.Json.JsonWrapper.SerializeObject(Timestamp, PlayFabUtil.ApiSerializerStrategy);
+            var json = PlayFab.Json.JsonWrapper.SerializeObject(Timestamp, PlayFab.Json.SimpleJsonInstance.ApiSerializerStrategy);
```

Comment:
Remove PlayFab.Json.SimpleJsonInstance.ApiSerializerStrategy parameter.
It's automatically used if you omit that parameter, and it FAILS if they swap out the serializer to Newtonsoft or something.

Reply:
(None of OUR code should ever provide that parameter)